tests/harvest/test_scenarios.py

```
@@ -159,19 +160,29 @@
         )
 
 
+@pytest.fixture
+def create_harvest_service(
+    store_info_sets: Any,
```

Ruff

Dynamically typed expressions (typing.Any) are disallowed in `store_info_sets`

(ANN401)